docs/MO_DG/prepare_model/MO_Python_API.md

```diff
@@ -0,0 +1,86 @@
+## Model Optimizer Python API {#openvino_docs_MO_DG_Python_API}
+
+Model Optimizer (MO) has a Python API for model conversion, which is represented by the `convert_model()` method in the openvino.tools.mo namespace.
+  `convert_model()` has all the functionality available from the command-line tool, plus the ability to pass Python objects (models, extensions) to `convert_model()` directly from memory without leaving the training environment (Jupyter Notebook or training scripts).
+  `convert_model()` returns an openvino.runtime.Model object which can be compiled and inferred or serialized to IR.
+
+```sh
+from openvino.tools.mo import convert_model
+
+ov_model = convert_model("resnet.onnx")
+```
+
+MO Python API allows the conversion of PyTorch models.
+
+```sh
+import torchvision
+
+model = torchvision.models.resnet50(pretrained=True)
+ov_model = convert_model(model, input_shape=[1,3,100,100])
+```
+
+The following types are supported as an input model for `convert_model()`:
+
+* PyTorch - `torch.nn.Module`, `torch.jit.ScriptModule`, `torch.jit.ScriptFunction`. Refer to the [Converting a PyTorch Model](@ref openvino_docs_MO_DG_prepare_model_convert_model_Convert_Model_From_PyTorch) article for more details.
+* TensorFlow/ TensorFlow2 / Keras - `tf.keras.Model`, `tf.keras.layers.Layer`, `tf.compat.v1.GraphDef`, `tf.Module`, `tf.compat.v1.wrap_function`, `tf.compat.v1.session`, `tf.train.checkpoint`, `tf.python.training.tracking.base.Trackable`(with limitations). Refer to the [Converting a TensorFlow Model](@ref openvino_docs_MO_DG_prepare_model_convert_model_Convert_Model_From_TensorFlow) article for more details.
```

Comment:
Please understand the current speed of conversion for objects loaded directly from Python. It turns out there is a regression in the speed comparing to earlier commits in master branch making this feature unusable. If degradation is confirmed, please remove this stuff from documentation.

Reply:
For 2022.3 branch I added documentation only for phase 1 due to regression:  https://github.com/openvinotoolkit/openvino/pull/14696

Phase 2 documentation in this PR will go to next release.